lexer_test.go

```diff
@@ -22,11 +22,20 @@ func TestValidKeyGroup(t *testing.T) {
 }
 
 func TestNestedQuotedUnicodeKeyGroup(t *testing.T) {
-	testFlow(t, `[ j . "ʞ" . l ]`, []token{
+	testFlow(t, `[ j . "ʞ" . 'l' ]`, []token{
```

Comment:
Why the change in that test?  the bare `l` should be valid right?

Reply:
It is for testing at the same time bare keys, normal string keys and literal string keys.

Reply:
Oh I see what you mean. In general I'm hesitant to modify existing tests to avoid inadvertent regressions, but in that case you're right it should have worked.